scipy/_lib/tests/test__util.py

```diff
@@ -56,6 +56,13 @@ def test_check_random_state():
     rsi = check_random_state(None)
     assert_equal(type(rsi), np.random.RandomState)
     assert_raises(ValueError, check_random_state, 'a')
+    try:
```

Comment:
this would have silently passed with the wrong `AttributeError` being caught I think. better to change to checking for numpy version >= 1.17.0 ?

Reply:
I would do `if hasattr(np.random, 'Generator'):` rather than a version check.

Reply:
Changed to a `hasattr` check and force-pushed.